### DISCUSSION TITLE
more scan lists?

### DISCUSSION THREAD

**Discussion by patomoi** [egzumer/uv-k5-firmware-custom#138] [Ideas]
hi, is possible to add more scan lists or they are limited by hardware or hard coded etc?. i feel in my case that only 2 scanlist are not sufficent.

**Comment by ErikS-web** [comment]
Already thought of this? More space for scan list means LESS space for the firmware?

I think it is more convenient to create multiple setting files with Chirp and then load the one, you want to apply.

**Comment by AubsUK** [comment]
I have just implemented this on the first version of firmware and CHIRP driver over on my repo - https://github.com/AubsUK/uv-k5-firmware-custom

The current main changes to Egzumer's firmware are:

* The two ScanLists have been removed and replaced with 10 ScanLists (0-9)
* When scanning, the ScanLists now show along the top of the screen
* Toggling ScanLists on/off is as simple as pressing the number
* Memory channels can now be excluded (LockOut/Skip) from scanning
* Scanning can be set to automatically start when the radio is turned on
* All the changes can be managed through CHIRP

You WILL need a custom driver to use CHIRP.  All instructions provided.

Open a discussion there if you have any questions or suggestions.
